src/gsTensor/gsTensorDomainBoundaryIterator.h

```diff
@@ -0,0 +1,324 @@
+/** @file gsTensorDomainBoundaryIterator.h
```

Comment:
This file should not be here (file moved to folder src/gsDomain in the latest stable)